lib/enumerate_it/base.rb

```diff
@@ -21,6 +21,10 @@ def list
         sorted_map.map { |_k, v| v.first }
       end
 
+      def collection
```

Comment:
I think we should call it `to_h`, what do you think?

```suggestion
      def to_h
```